README.md

```diff
@@ -30,71 +30,20 @@ For more context, starts with this [stackoverflow thread](http://stackoverflow.c
 
 In order to use this token acquisition method, you need to configure a service principal. Please follow [this walkthrough](https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/).
 
-See the [sample](./sample/client_credentials_sample.py).
-```python
-import adal
-
-context = adal.AuthenticationContext('https://login.microsoftonline.com/ABCDEFGH-1234-1234-1234-ABCDEFGHIJKL')
-RESOURCE = '00000002-0000-0000-c000-000000000000' #AAD graph resource
-token = context.acquire_token_with_client_credentials(
-    RESOURCE,
-    "http://PythonSDK", 
-    "Key-Configured-In-Portal")
-```
+See the [sample](sample/client_credentials_sample.py#L46-L53).
```

Comment:
Yugang has sharp eyes! :-)

Actually I intentionally included the line range as a middle ground. Without them serving as highlighted code snippet, the full sample contains extra code and looks less straightforward than the snippets, such as the website sample. Those line ranges will easily go of date for sure, but at that time they will at least still point to the right file.

I will still follow your suggestion to call out some comment text, in my next commit. Thanks and stay tuned.